models/User.js

```diff
@@ -3,7 +3,7 @@ var crypto = require('crypto');
 var mongoose = require('mongoose');
 
 var userSchema = new mongoose.Schema({
-  email: { type: String, unique: true, lowercase: true },

```

Comment:
Email addresses are no longer stored as lowercase in the DB.
Case is maintained since technically SumTotal/Skillsoft email addresses contain capitalization.
This should help to fix a small bug causing Teams links to not function as seamlessly as possible.